.mailmap

```diff
@@ -19,3 +19,4 @@ Chris Dituri <csdituri@gmail.com> chris dituri
 Ole Herman S. Elgesem <ole@northern.tech> <ole.elgesem@northern.tech> <ole.elgesem@cfengine.com>
 Vratislav Podzimek <vratislav.podzimek@northern.tech> <vratislav.podzimek@cfengine.com>
 Craig Comstock <craig.comstock@northern.tech> <craig.comstock@cfengine.com>
+Trix Farrar <trix@basement.net> <trix@doyle.basement.net>
```

Comment:
trix@doyle.basement.net address is not valid.  s/b

Trix Farrar <trix@basement.net> <trixfarrar@gmail.com>

Reply:
> > You want `John D. "Trix" Farrar` or just Trix Farrar?
> I guess `John D. "Trix" Farrar`, since that's how I'm credited in the pkg module.

Well, we can align them in the .mailmap if you prefer one over another.